cvpysdk/subclients/vssubclient.py

```diff
@@ -240,359 +228,135 @@ def _process_restore_request(self, vm_names, restore_content):
 
-            Returns:
-                list - list of all folders or files with their full paths inside the input path
+                Keyword argument of browse options 
+                    Example - vm_files_browse( 
+                        path='\\vmname\\', show_deleted=True, to_time='2016-04-31 12:00:00'
+                    )
 
-                dict - path along with the details like name, file/folder, size, modification time
+                Refer Backupset.default_browse_options for all the supported options
 
-            Raises:
-                SDKException:
-                    if failed to browse content
-
-                    if response is empty
+        Returns:
+            list - List of only the file, folder paths from the browse response
+            dict - Dictionary of all the paths with additional metadata which are retrieved from browse
 
-                    if response is not success
         """
-        return self.browse(vm_path, show_deleted_files, True)
-
-    def vm_files_browse_in_time(
-            self,
-            vm_path='\\',
-            show_deleted_files=True,
-            restore_index=True,
-            from_date=None,
-            to_date=None):
-        """Browses the Files and Folders of a Virtual Machine in the time range specified.
-
-            Args:
-                vm_path             (str)   --  folder path to get the contents of
-                    default: '\\'; returns the root of the Backup content
-
-                show_deleted_files  (bool)  --  include deleted files in the content or not
-                    default: True
-
-                restore_index       (bool)  --  restore index if it is not cached
-                    default: True
-
-                from_date           (str)   --  date to get the contents after
-                        format: dd/MM/YYYY
-
-                        gets contents from 01/01/1970 if not specified
-                    default: None
-
-                to_date             (str)  --  date to get the contents before
-                        format: dd/MM/YYYY
-
-                        gets contents till current day if not specified
-                    default: None
-
-            Returns:
-                list - list of all folders or files with their full paths inside the input path
-
-                dict - path along with the details like name, file/folder, size, modification time
-
-            Raises:
-                SDKException:
-                    if from date value is incorrect
-
-                    if to date value is incorrect
-
-                    if to date is less than from date
 
-                    if failed to browse content
-
-                    if response is empty
-
-                    if response is not success
-        """
-        return self.browse_in_time(
-            vm_path, show_deleted_files, restore_index, True, from_date, to_date
-        )
+        return self.browse(*args, **kwargs)
```

Comment:
Before browse, the value for vm_disk_browse, should be set to True in the options Dict